study.md

```diff
@@ -52,13 +53,13 @@ help manage view-state?
 What are the advantages of a single page application over a traditional appliction?
 
 ```md
-<!-- your answer here -->
+The code is easier to maintain
 ```
 
 ## List some technologies used to make SPA's
 
 What are some of the tools used to create single page applications?
 
 ```md
-<!-- your answer here -->
+ajax and json
```

Comment:
These are key but also front end frameworks like Ember, Angular, and React are very vital to developing a SPA.